Resources/Prototypes/Entities/Items/tools.yml

```diff
@@ -111,3 +111,24 @@
 
   - type: Item
     sprite: Objects/Tools/multitool.rsi
+
+- type: entity
+  name: RCD
+  parent: BaseItem
+  id: RCD
+  description: An advanced construction device which can place floors / walls down quickly.
+  components:
+  - type: RCD
+  - type: UseDelay
+    delay: 2.0
+    
+  - type: Sprite
```

Comment:
I was refering to all the components, only use the separation when writting a new prototype (see: https://hackmd.io/Awj5TvdXTaOmhU2XZOsdQg)